Procfile

```diff
@@ -0,0 +1 @@
+web: nodemon index.js
```

Comment:
By creating this file and adding the`nodemon` dependency, if you run `heroku local` now, it well run a localserver that picks up on local changes and update the node server automatically.